### ISSUE TITLE
Add Failed: Invalid Role

### ISSUE THREAD

**Issue by lrhazi** [taers232c/GAMADV-XTD#15]
I noticed that if I try to add an invalid email address I get "Invalid Role" error.. is that from Google API ?

```
[root@694ee29874b3 app]# gam update group test@cua.edu add member user kdjskjkjkjaskjf@yahoo.comhdhd
Group: test@cua.edu, Add 1 Member
  Group: test@cua.edu, Member: kdjskjkjkjaskjf@yahoo.comhdhd, Add Failed: Invalid Role
[root@694ee29874b3 app]#
```

**Comment by taers232c** [comment]
Thank you for pointing this out, it was my mistake, corrected in 4.57.04.

Ross

**Comment by lrhazi** [comment]
Thanks!
